ramalama

```diff
@@ -135,11 +136,10 @@ def init_pull(repos_ollama, manifests, accept, registry_head, model_name, model_
 
 
 def huggingface_download(ramalama_store, model, directory, filename):
-    return run_cmd(["huggingface-cli", "download", directory, filename, "--cache-dir", ramalama_store + "/repos/huggingface/.cache", "--local-dir", ramalama_store + "/repos/huggingface"])
+    return run_cmd(["huggingface-cli", "download", directory, filename, "--cache-dir", ramalama_store + "/repos/huggingface/.cache", "--local-dir", ramalama_store + "/repos/huggingface/" + directory])
 
 
 def try_huggingface_download(ramalama_store, model, directory, filename):
-    huggingface_download(ramalama_store, model, directory, filename)
```

Comment:
Actually I missed this, this may be a bug.

The first time you download the model the stdout content is much more than just a filename

Reply:
Actually seems like it works anyway, nevermind